content/community.json

```diff
@@ -69,24 +69,7 @@
       "pictureUrl": "/img/community/ugc/codecentric.png"
     }
   ],
-  "events": [
-    {
-      "url": "https://www.widsconference.org/",
-      "title": "Women in Data Science San Diego",
-      "description": "Elle O'Brien is talking about how to share data science projects.",
-      "city": "Virtual event",
-      "date": "2020-06-09",
-      "pictureUrl": "/img/community/events/WiDS_logo.jpg"
-    },
-    {
-      "url": "https://mlopsworld.com/",
-      "title": "MLOps: Production and Engineering World",
-      "description": "Elle O'Brien is speaking about adapting CI/CD to ML.",
-      "city": "Virtual event",
-      "date": "2020-06-17",
-      "pictureUrl": "/img/community/events/toronto_ml.jpg"
```

Comment:
remove images if they are not used anymore? Should we just keep one of the events (or all of them) as an example? I'm not sure we should even log warnings on expired events to be honest. It's an intended behavior

Reply:
Good point on the images. If we don't want the logging behavior though, I can remove it.

Reply:
@rogermparent up to you to decide on this :)